src/core/style/_variables.scss

```diff
@@ -18,6 +18,11 @@ $pi: 3.14159265;
 // Easing Curves
 // TODO(jelbourn): all of these need to be revisited
 
+// The default animation curves used by material design.
+$linear-out-slow-in-timing-function: cubic-bezier(0.0, 0.0, 0.2, 0.1) !default;
```

Comment:
Can you prefix these variables with `md-`? (it's a bug that the existing ones don't have a prefix)